src/util/include/util/DispatchPool.h

```diff
@@ -39,6 +39,21 @@ class DispatchPool final {
             v->on(std::forward<Args>(args)...);
         }
     }
+    
+    /**
+     * @brief Invokes the `deleteOn()` method of all registered `ListenerT`s.
+     * The deleteOn() is supposed to delete the listener
+     * (or at the very least remove any reference to the dispatcher in the listener)
+     */
+    template <typename... Args>
+    void dispatchAndClear(Args&&... args) const {
+        // We cannot iterate on the this->pool: the listener's deletion would trigger calls to DispatchPool::remove()
+        // leading to possible segfaults.
+        auto p = std::move(this->pool);
+        for (auto* v: p) {
+            v->deleteOn(std::forward<Args>(args)...);
```

Comment:
Unless I'm missing something, shouldn't `deleteOn` also be on the list of required methods in the class's doc comment?